src/Move.py

```diff
@@ -23,32 +22,44 @@ def __init__(self, piece: Piece, newPos: C, pieceToCapture: Optional[Piece] = No
         self.newPos = newPos
         self.pieceToCapture = pieceToCapture
         # For en passant and castling
-        self.specialMovePiece = None  # TODO: this should be a 'Piece' type to satisfy mypy
+        self.specialMovePiece = (
+            None  # TODO: this should be a 'Piece' type to satisfy mypy
+        )
         # For castling
         self.rookMove = None  # TODO: this should be a 'Move' type to satisfy mypy
 
     def __str__(self) -> str:
-        displayString = 'Old pos : ' + str(self.oldPos) + \
-                        ' -- New pos : ' + str(self.newPos)
+        displayString = (
+            'Old pos : ' + str(self.oldPos) + ' -- New pos : ' + str(self.newPos)
+        )
         if self.notation:
             displayString += ' Notation : ' + self.notation
         if self.passant:
-            displayString = 'Old pos : ' + str(self.oldPos) + \
-                            ' -- New pos : ' + str(self.newPos) + \
-                            ' -- Pawn taken : ' + str(self.specialMovePiece)
+            displayString = (
+                'Old pos : '
+                + str(self.oldPos)
+                + ' -- New pos : '
+                + str(self.newPos)
```

Comment:
This part is obviously repeated. A tool can't detect that, but please change it so it doesn't repeat itself.